hokusai/staging.yml

```diff
@@ -93,6 +93,17 @@ spec:
                 operator: In
                 values:
                 - foreground
+                - foreground-spot
+      topologySpreadConstraints:
+      - maxSkew: 1
```

Comment:
when spreading pods between two groups, the max difference (in pod count) between the groups is 1.